src/components/molecules/TaskItem.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+	<div class="card">
+		<div class="card-body">
+			<base-image
+				v-if="image"
+				:img-src="image"
+				class="image"
+				role="presentation"
+			/>
```

Comment:
We can pass $attrs which will hold `imgSrc` and `fill` properties which will be passed to the parent component. This way we can write e.g. `<task-item fill="purple" imgSrc="<some_svg_path>"></task-item>` to assing an SVG and a color to the SVG component.

```suggestion
			<base-image
				v-if="$attrs.imgSrc"
				v-bind="$attrs"
				class="image"
				role="presentation"
			/>
```

Reply:
Done :-)